Dockerfile

```diff
@@ -16,32 +18,21 @@ RUN apt-get update && \
                        python-numpy \
                        python-toml \
                        python-yaml \
-                       unzip
-
-RUN curl -ssL http://get.gazebosim.org > /tmp/install_gazebosim.sh
-RUN sed -i "s/GZ_VER=9/GZ_VER=8/" /tmp/install_gazebosim.sh
-RUN chmod +x /tmp/install_gazebosim.sh
-RUN /tmp/install_gazebosim.sh
+                       unzip \
+                       gazebo9 \
```

Comment:
Yes it's nicer like this!

Reply:
Yes that was easy.

Reply:
Until we move to gazebo10 and stay on Ubuntu 18 :sweat_smile: